README.md

```diff
@@ -138,6 +138,28 @@ end
 [ Info: Yo Dawg! it is all good
 ```
 
+### Respecting `maxlog` convention
+
+An `ActiveFilterLogger` can be used to wrap another logger to obey `maxlog` directives, for example,
+similar to the `make_throttled_logger` example below,
+```julia
+function make_maxlog_logger(logger)
+    counts = Dict{Symbol,Int}()
+    return ActiveFilteredLogger(logger) do log
+        haskey(log.kwargs, :maxlog) || return true
+        if !haskey(counts, log.id) || (counts[log.id] < log.kwargs[:maxlog])
+            # then we will log it, and update the corresponding count
+            counts[log.id] = get(counts, log.id, 0) + 1
+            return true
+        else
+            return false
+        end
```

Comment:
How about this (I found it slightly easier to read):
```suggestion
        maxlog = get(log.kwargs, :maxlog, nothing)
        maxlog === nothing && return true # no limit
        if (c = get(counts, log.id, 0); c < maxlog)
            # log this message and update the count
            counts[log.id] = c + 1
            return true
        else
            return false
        end
```

Reply:
this is nicer, thanks